data/json/monsters.json

```diff
@@ -5354,7 +5402,8 @@
       {
         "type": "bite",
         "cooldown": 5
-      }
+      },
+      [ "scratch", 20 ]
```

Comment:
Grabbers having scratch makes them delay the grabbing, which is supposed to be their primary function.
Though I guess so does bite.